programs/ziti-edge-tunnel/config-utils.c

```diff
@@ -138,3 +148,28 @@ void update_identity_config(uv_loop_t *l, const char *identifier, const char *cf
     }
 }
 
+char* resolve_directory(const char* path) {
```

Comment:
this can/should be `static` since it's not referenced outside of this module. also whitespace.